Blueprints/BlueprintDefinitions/vs2022/MessageProcessingFramework/template/src/BlueprintBaseName.1/serverless.template

```diff
@@ -0,0 +1,64 @@
+{
+  "AWSTemplateFormatVersion": "2010-09-09",
+  "Transform": "AWS::Serverless-2016-10-31",
+  "Description": "AWS Message Processing Framework for .NET Template. This template is partially managed by Amazon.Lambda.Annotations (v1.2.0.0).",
+  "Resources": {
+    "MessageProcessingFrameworkDemoQueue": {
+      "Type": "AWS::SQS::Queue"
+    },
+    "BlueprintBaseName1HandlerGenerated": {
+      "Type": "AWS::Serverless::Function",
+      "Metadata": {
+        "Tool": "Amazon.Lambda.Annotations"
+      },
+      "Properties": {
+        "Runtime": "dotnet8",
+        "CodeUri": ".",
+        "MemorySize": 256,
```

Comment:
Should this be 512?

Reply:
This is being set by Lambda Annotations, which we want to update its default value to 512.

I could override this specific template via the attributes, but would rather just update once centrally.